server/db/dbManager.ts

```diff
@@ -17,6 +19,11 @@ export interface FilterObject {
 	search: string;
 }
 
+export interface PublicUserData{
```

Comment:
This type of mistake means that eslint doesn't cover this file.
```suggestion
export interface PublicUserData {
```

Reply:
Actually, it looks like the "space-before-blocks" rule doesn't apply to interfaces because that wouldn't pass for all other brackets I am using in this file. I am not sure why this is the case I will try to find a fix.

Reply:
I assume the tslint can fix it, not eslint. Am I right?

Reply:
looks like the eslint-typescript plugin does not have a fix for this https://github.com/typescript-eslint/typescript-eslint/issues/1606. I will use prettier in addition to eslint.